### PR TITLE
Agms 554

### PR DESCRIPTION
**Pull request by finp**
prep showcase for downstream
remove all but cordova apps
rename cordova to ionic
insert version no

### CODE REVIEW

File: modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc [hunk 1]
```diff
@@ -14,31 +14,47 @@ Showcase apps demonstrate many features of {product-name}. To see these features
...
 
+// tag::excludeDownstream[]
 . To demonstrate {metrics-service}, xref:mobile-metrics.adoc#provisioning-metrics-service[Provision the {metrics-service} service] For more information about features, see xref:mobile-metrics.adoc[{metrics-service} Service]
+// end::excludeDownstream[]
 
 . To demonstrate {keycloak-service}:
 
-.. xref:identity-management.adoc#provisioning-the-identity-management-service[Provision the {keycloak-service} service].
+// tag::excludeDownstream[]
+.. xref:{identity-management.adoc}#provisioning-the-identity-management-service[Provision the {keycloak-service} service].
```
Comment (jstaffor) [inline modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]: Use of xref upstream?

File: modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc [hunk 2]
```diff
@@ -14,31 +14,47 @@ Showcase apps demonstrate many features of {product-name}. To see these features
...
 
-.. Set credentials for the created user. Follow the steps link:https://www.keycloak.org/docs/3.3/server_admin/topics/users/credentials.html[here].
+.. Set credentials for the created user. Follow the steps link:{idm-docs-link}[documentation].
 
 +
-For more information about features, see xref:identity-management.adoc[{keycloak-service} Service]
+For more information about features, see link:{idm-guide-link}[{keycloak-service} Service]
+
+// tag::excludeDownstream[]
+. To demonstrate {unifiedpush-service}xref:push-notifications.adoc#provisioning-push-notifications-service[Provision the {unifiedpush-service} service]. For more information about features, see xref:push-notifications.adoc[{unifiedpush-service} Service]
```
Comment (jstaffor) [inline modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]: Use of xref upstream?

File: modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc [hunk 3]
```diff
@@ -14,31 +14,47 @@ Showcase apps demonstrate many features of {product-name}. To see these features
...
+// end::excludeDownstream[]
+
+// tag::excludeUpstream[]
+. To demonstrate {unifiedpush-service} the see link:{push-guide-link}[{unifiedpush-service} Service]
```
Comment (jstaffor) [inline modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]:
```suggestion
. To demonstrate {unifiedpush-service}, see link:{push-guide-link}[{unifiedpush-service} Service]
```

File: modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc [hunk 4]
```diff
@@ -215,31 +234,32 @@ Xamarin::
...
 --
-// end::excludeDownstream[]
+////
 ====
 
 [#binding]
 == Binding the Showcase {mobile-client} to {product-name}
 
+// tag::excludeDownstream[]
 . xref:mobile-metrics.adoc#binding-a-mobile-client-with-the-metrics-service[Bind the {mobile-client} to the {metrics-service} service] if you want to demonstrate the {metrics-service} service.
```
Comment (jstaffor) [inline modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]: Use of xref upstream?

File: modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc [hunk 5]
```diff
@@ -215,31 +234,32 @@ Xamarin::
...
 = Downloading the Mobile Services Configuration File
 
-NOTE: This section only applies to the Ionic platform.
-
 . Open your {mobile-client} in Mobile Developer Console.
 . Copy the `mobile-services.json` configuration to your clipboard.
  Save the contents of the clipboard to a new file called `mobile-services.json`.
 +
-NOTE: The mobile-services.json file is the link between your provisioned services on OpenShift and the mobile app you are developing. This file provides all required configuration to initialise the various SDKs and get them hooked up/connected to the back-end services.
+NOTE: The mobile-services.json file is the link between the mobile services running on OpenShift and the mobile app you are developing. This file provides all required configuration to initialise the various SDKs and get them hooked up/connected to the back-end services.
```
Comment (jstaffor) [inline modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]:
```suggestion
NOTE: The mobile-services.json file is the link between the mobile services running on OpenShift and the mobile app you are developing. The mobile-services.json file provides all required configuration to initialise the various SDKs and connect them to the back-end services.
```

File: modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc [hunk 6]
```diff
@@ -215,31 +234,32 @@ Xamarin::
...
 +
 . Copy the contents of the `mobile-services.json` file and paste them into the `mobile-services.json` located in `src`.
```
Comment (jstaffor) [inline modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]:
```suggestion
. Copy the contents of the `mobile-services.json` file into the `mobile-services.json` file located in `src`.
```

File: modules/ROOT/pages/showcase-apps-all.adoc [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+
+// tag::excludeDownstream[]
+[#certs]
+
+include::_partials/proc_self-signed-certs.adoc[leveloffset=1,tags={include-tags}]
+// end::excludeDownstream[]
+
+[#features]
+== Demonstrating the ShowCase App Features
```
Comment (jstaffor) [inline modules/ROOT/pages/showcase-apps-all.adoc]:
```suggestion
== Demonstrating the ShowCase App features
```

File: modules/ROOT/pages/_partials/identity-management-demo.adoc [hunk 8]
```diff
@@ -4,19 +4,19 @@
 
 NOTE: Please make sure you have https://www.keycloak.org/docs/3.3/server_admin/topics/users/create-user.html[created a user] and https://www.keycloak.org/docs/3.3/server_admin/topics/users/credentials.html[set up credentials] first.
```
[the remark this answers is not in the document]
Comment (finp) [reply modules/ROOT/pages/_partials/identity-management-demo.adoc]: rewritten as:

.Prerequisite

Make sure you have created a user and set up credentials for that user as describe in the link:{idm-docs-link}[{idm-name} Documentation].

File: modules/ROOT/pages/_partials/showcase-apps/index.adoc [hunk 9]
```diff
@@ -1,9 +1,8 @@
 
 
-= Showcase Apps
+= About the Showcase App
```
[the remark this answers is not in the document]
Comment (finp) [reply modules/ROOT/pages/_partials/showcase-apps/index.adoc]: not a proc

**Pull request merged**